resources/css/index.css

```diff
@@ -19,193 +18,167 @@ body, html {
+  margin-left: -5px;
 }
 
 .container {
-  position: relative;
-  z-index: 2;
+  background-attachment: fixed;
+  background-image: url(../images/background/Bg.png);
+  background-position: 50% 0px;
+  height: 750px;
+  width: 1600px;
 }
 
-.layout-positioner h1 a {
-  text-decoration: none;
-  color: #fff;
+.divider {
+  display: inline-block;
+  margin-top: 30px;
 }
 
-.layout-positioner {
-  z-index: 3;
+.logo {
+  float: left;
+  margin-top: 35px;
+  margin-left: 175px;
 }
 
-.layout-positioner {
-  margin: 0px auto;
-  margin-bottom: 40px;
-  width: 1250px;
+.logo-title {
+  margin: 0;
 }
 
-.layout-positioner .header {
-  margin: 35px 0;
+.logo-link {
+  color: #fff;
+  font-family: "Yeseva One", sans-serif;
+  font-size: 36px;
+  font-weight: 400;
+  text-decoration: none;
+  text-transform: uppercase;
 }
 
-.layout-positioner::after {
-  content: "";
-  display: table;
-  clear: both;
- }
-
- .navbar {
-  float: right;
+.nav {
+  float: left;
+  margin-left: 210px;
  }
 
- .navbar .main-menu {
+.main-menu {
   list-style: none;
   margin: 0;
   padding: 0;
  }
 
- .navbar .main-menu li {
+.main-menu .menu-button {
+  box-sizing: border-box;
   display: inline-block;
   margin:0px;
-  box-sizing: border-box;
   margin-left: -4px;
  }
 
- .navbar .main-menu li:first-child {
+ .main-menu .menu-button:first-child {
   margin-left: 0;
  }
 
- .navbar .main-menu li a {
-  display: block;
-  text-decoration: none;
+.menu-button .menu-link {
   color: #fff;
+  display: block;
   font-family: "GenB", sans-serif;
   font-size: 21px;
-  padding: 44px 30px;
   overflow: hidden;
+  padding: 44px 30px;
+  text-decoration: none;
  }
 
- .navbar .main-menu li:hover a {
+ .main-menu .menu-button:hover a {
   border-top: 5px solid #c9c6ba;
   padding-top: 39px;
  }
 
- .social-media {
-  float: right;
+ .header::after {
+  clear: both;
+  content: "";
+  display: table;
+ }
+
+.header {
+  position: relative;
+  z-index: 2;
+}
+
+ .socials {
+  float: left;
  }
 
- .social-media .social-menu {
+ .socials .social-menu {
   list-style: none;
   margin: 0;
   padding: 0;
  }
 
- .social-media .social-menu li {
+ .social-button {
+  box-sizing: border-box;
   display: inline-block;
   margin:0px;
-  box-sizing: border-box;
   margin-left: -4px;
   padding: 36px 0;
  }
 
- .social-media .social-menu li a {
-  font-size: 21px;
+.social-button .social-link {
   color: #dab33e;
   display: block;
-  text-decoration: none;
+  font-size: 21px;
   padding: 8px 5px;
+  text-decoration: none;
  }
 
- .social-media social-menu li a:hover {
+ .social-button .social-link:hover {
   color: #f0c955;
   text-shadow: 1px 1px 2px #000;
  }
 
- .social-media .social-menu li:first-child {
+ .social-button:first-child {
   margin-left: 0px;
  }
+
+ .shadow {
+  box-shadow: 0px 50px 119px 0px rgba(0,0,0,0.67);
+  height: 160px;
+  position: absolute;
+  top: -160px;
+  width: 1600px;
+}
+
+.intro {
+  text-align: center;
+  margin-top: 110px;
```

Comment:
Seems like parent component's padding should do this job

Reply:
fixed.